packages/scripts/scripts/lint-js.js

```diff
@@ -11,12 +11,15 @@ const {
 	fromConfigRoot,
 	getCliArgs,
 	hasCliArg,
+	hasFileInCliArgs,
 	hasPackageProp,
 	hasProjectFile,
 } = require( '../utils' );
 
 const args = getCliArgs();
 
+const defaultFilesArgs = ! hasFileInCliArgs ? [ '.' ] : [];
```

Comment:
Should we use `./src` instead of `.` to align with `wp-scripts build`?

Reply:
In #15977, I propose we ignore `build` and `node_modules` folders by default, so this should be less important than at the moment.